apps/api/src/app/widgets/e2e/mark-as-by-mark.e2e.ts

```diff
@@ -0,0 +1,201 @@
+import { expect } from 'chai';
+import axios from 'axios';
+
+import { UserSession } from '@novu/testing';
+import {
+  MessageEntity,
+  MessageRepository,
+  NotificationTemplateEntity,
+  SubscriberEntity,
+  SubscriberRepository,
+} from '@novu/dal';
+import { ChannelTypeEnum, MarkMessagesAsEnum } from '@novu/shared';
+
+describe('Mark as Seen - /widgets/messages/mark-as (POST)', async () => {
```

Comment:
missing same tests for the subscribers controller ;)

Reply:
in the end, i will need to duplicate the tests, was not sure if to do it or not, but we do need to make sure that the subscriber controller works as expected. i will create the same tests there as well 💪